### DISCUSSION TITLE
云端同步设置

### DISCUSSION THREAD

**Discussion by myaijarvis** [the1812/Bilibili-Evolved#3096] [功能建议 / Ideas]
我有一个建议，就是把自己装的插件和插件的一些设置变成json格式的文件上传到云端，这样多设备用的时候就可以同步了，非常方便。
希望作者可以考虑一下，类似于油猴脚本可以使用onedrive或者其他途径进行备份的功能。

**Comment by Geeyun-JY3** [comment]
其实，为什么不在脚本管理器里干这事

**Reply by FoundTheWOUT** [reply]
用过的都说烂（逃

**Reply by Geeyun-JY3** [reply]
烂在哪（我其实一直有在用，备份到 WebDAV，还行吧

**Reply by FoundTheWOUT** [reply]
总觉得同步各种怪怪，无论是脚本本身还是脚本数据，曾一度把我的配置搞得一团乱，也许是我不太会用吧

**Reply by Geeyun-JY3** [reply]
跟随浏览器多端同步好不好用不好说，没怎么用过，我是备份一整个 zip 到 WebDAV 放一份。

**Comment by the1812** [comment]
#1749
